pkg/kinflate/app/application.go

```diff
@@ -39,8 +47,102 @@ func New(loader loader.Loader) (Application, error) {
 }
 
 // Resources computes and returns the resources from the manifest.
-func (a *applicationImpl) Resources() []resource.Resource {
+func (a *applicationImpl) Resources() (types.ResourceCollection, error) {
 	// This is where all the fun happens.
 	// From the manifest create the resources.
-	return nil
+	m, err := newManifest(a.loader)
+	if err != nil {
+		return nil, err
+	}
+	a.manifest = *m
+
+	errs := &interror.ManifestErrors{}
+	allResourcesCollection, err := rescourceCollectionFromManifest(a.loader, m)
+	if err != nil {
+		return allResourcesCollection, err
+	}
+
+	for _, pkgPath := range m.Packages {
```

Comment:
May be we can have an internal method called "subAppResources()" to get all the subAppResources ?